The Azure AD role is misconfigured it only works if the permissions a…

NorbertoGil opened the pull request:
…re set to SAML Single Sign-on and the Single Sign-on option is NOT Selected

Azure AD SSO Role in the permissions in the bottom, the option needs to be SAML Single Sign-on and not SETUP Single Sign-on. Also the Single Sign on only option needs to be unchecked

azurecla commented:
Hi __@jinmiaoms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@SuperScottz 
Please review and approve or disapprove this pull request with a comment here. Thanks.

rmca14 commented:
@superscottz 
Please review this pull request and approve or disapprove with a comment here. Thanks.

deneha commented:
@jinmiaoms This request has merge conflicts.  Would you please fix the merge conflict?

SuperScottz commented:
I have created another pull request to merge the change from this one, let's close this.
https://github.com/Azure/azure-content-pr/pull/15636

@jinmiaoms Thank you for decent feedback!

NorbertoGil committed:
The Azure AD role is misconfigured it only works if the permissions are set to SAML Single Sign-on and the Single Sign-on option is NOT Selected

Azure AD SSO Role in the permissions in the bottom, the option needs to be SAML Single Sign-on and not SETUP Single Sign-on. Also the Single Sign on only option needs to be unchecked

NorbertoGil committed:
Update active-directory-saas-netsuite-tutorial.md
